samples/dotnet/KernelHttpServer/SemanticKernelEndpoint.cs

```diff
@@ -105,31 +105,31 @@ public SemanticKernelEndpoint(IMemoryStore memoryStore)
 
         var iterations = 1;
 
-        while (!result.Variables.ToPlan().IsComplete &&
-               result.Variables.ToPlan().IsSuccessful &&
-               iterations < maxSteps)
+        Plan? plan = null;
```

Comment:
Should we consider getting the Plan JSON directly in this endpoint instead of expecting the plan to be one of the ContextVariables?